pkg/apply/destroyer.go

```diff
@@ -70,6 +70,7 @@ func (d *Destroyer) Run() <-chan event.Event {
 	go func() {
 		defer close(ch)
 		infos, _ := d.ApplyOptions.GetObjects()
+		_ = prune.ClearGroupingObj(infos)
```

Comment:
Could we add comment that explains why this is needed here? Also, why are we ignoring the error?